meinberlin/apps/bplan/serializers.py

```diff
@@ -66,8 +66,8 @@ def create(self, validated_data):
         orga = orga_model.objects.get(pk=orga_pk)
         validated_data['organisation'] = orga
 
-        start_date = validated_data.pop('start_date')
-        end_date = validated_data.pop('end_date')
+        start_date = validated_data['start_date']
+        end_date = validated_data['end_date']
```

Comment:
nitpick: also `.get` to be consistent?

Reply:
Wait, there is no `get()` on lists - does it work in the change below? Is `validated_data` a dict there?

Reply:
Answer to myself: it is, otherwise `pop(foo, default)` wouldn't work - slightly confusing imho, but never mind :)